client/package.json

```diff
@@ -44,6 +56,19 @@
   },
   "devDependencies": {
     "@testing-library/jest-dom": "^5.11.9",
-    "@testing-library/react": "^11.2.3"
+    "@testing-library/react": "^11.2.3",
+    "@types/bootstrap": "^5.0.8",
+    "@types/react-bootstrap": "^0.32.25",
+    "@typescript-eslint/eslint-plugin": "^4.20.0",
+    "@typescript-eslint/parser": "^4.20.0",
+    "bootstrap": "^4.6.0",
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-react": "^7.23.1",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "react-bootstrap": "^1.5.1",
+    "typechain": "^4.0.2"
```

Comment:
`typechain` is listed in both `dependencies` and `devDependencies`. Can we take it out of one of them (take out of `dependencies` if we only need it outside of prod, otherwise take it out of `devDependencies`)? Same with `@types/bootstrap` and a few others.